2.crud_queries/tasks.md

```diff
@@ -1,38 +1,80 @@
 1. Create a database named `blog`.
 
+use blog
+
 2. Create a collection called 'articles'.
 
+db.createCollection('articles')
+
 3. Insert multiple documents(at least 3) into articles. It should have fields
```

Comment:
Alternatively you can use insertMany to insert multiple documents at the sam time.
```js
db.articles.insertMany([{}, {}, {}])
// each object represents an article here.
```